packages/social/src/google-provider.service.ts

```diff
@@ -37,7 +38,8 @@ export class GoogleProvider extends AbstractProvider<GoogleAuthParams, never> {
   getUserInfoFromTokens(tokens: SocialTokens): object {
     try {
       const encodedPayload = tokens.id_token.split('.')[1];
-      const decodedPayload = Buffer.from(encodedPayload, 'base64').toString('utf8');
+      const decodedPayload = Buffer.from(convertBase64urlToBase64(encodedPayload), 'base64')
```

Comment:
Node.JS does not officially support `base64url` encoding encoding even if `Buffer.from(base64urlStr, 'base64')` seems to largely work in most cases. This line adds more safety when decoding the payload.

I couldn't find a valid JSON encoded in base64url to make `Buffer.from(xxx, 'base64)` fails. This is why no new test has been added.